lib/rest_in_peace/response_converter.rb

```diff
@@ -42,7 +42,14 @@ def convert_from_array
     end
 
     def convert_from_hash(entity = body, instance = existing_instance)
-      instance.force_attributes_from_hash entity
+      entity_with_indifferent_access = entity.with_indifferent_access
+
+      # We received a validation error and we need to add it to the model.
+      if entity_with_indifferent_access[:status] == 422
+        instance.errors.add(:base, entity_with_indifferent_access[:message])
```

Comment:
Not every instance has `errors`, only the ones which have `acts_as_active_model` activated (check [here](https://github.com/ninech/REST-in-Peace/blob/24859-ipv6_errors_not_present/lib/rest_in_peace/definition_proxy.rb#L34) and [here](https://github.com/ninech/REST-in-Peace/blob/24859-ipv6_errors_not_present/lib/rest_in_peace/active_model_api.rb#L61-L63)).